__tests__/app.test.js

```diff
@@ -45,4 +45,51 @@ describe("app", () => {
         });
     });
   });
+  describe("/api/reviews", () => {
+    test("status: 200", () => {
+      return request(app).get("/api/reviews").expect(200);
+    });
+    test("responds with a reviews array", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body)).toBe(true);
+        });
+    });
+    test("all objects returned in array", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.length).toBe(13);
+        });
+    });
+    test("array contains objects with correct properties", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((review) => {
+            expect(review).toHaveProperty("owner", expect.any(String));
+            expect(review).toHaveProperty("title", expect.any(String));
+            expect(review).toHaveProperty("review_id", expect.any(Number));
+            expect(review).toHaveProperty("category", expect.any(String));
+            expect(review).toHaveProperty("review_img_url", expect.any(String));
+            expect(review).toHaveProperty("created_at", expect.any(String));
+            expect(review).toHaveProperty("votes", expect.any(Number));
+            expect(review).toHaveProperty("designer", expect.any(String));
+            expect(review).toHaveProperty("comment_count");
+          });
+        });
+    });
+    test("reviews sorted in descending order of date", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body[0]["created_at"]).toBe("2021-01-25T11:16:54.963Z");
+        });
```

Comment:
This is just tested the first returned value, the rest could be in the wrong order, I suggest trying out the [jest-sorted](https://www.npmjs.com/package/jest-sorted) library to test if the returned array's in the right order 🙂